### ISSUE TITLE
[Story navigation buttons] Navigation button's label on the last page of a story should read "next story"

### ISSUE THREAD

**Issue by gmajoulet** [ampproject/amphtml#33136]
Navigation button's label on the last page of a story still reads "last page", when it should read "next story".
This line should be replaced with a `NEXT_STORY` button state.

https://github.com/ampproject/amphtml/blob/644140058595f2e175df75807c67fd1e3afeb42b/extensions/amp-story/1.0/pagination-buttons.js#L337-L339

cc @ampproject/wg-stories
